src/core/evaluator.js

```diff
@@ -1415,9 +1415,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
 
           var glyphUnicode = glyph.unicode;
           var NormalizedUnicodes = getNormalizedUnicodes();
-          if (NormalizedUnicodes[glyphUnicode] !== undefined) {
-            glyphUnicode = NormalizedUnicodes[glyphUnicode];
-          }
+          glyphUnicode = normalizeUnicode(glyphUnicode, NormalizedUnicodes);
```

Comment:
I don't think `NormalizedUnicodes` is used elsewhere in this file, so it looks a bit strange to pass it in as a parameter since it's not really a variable. Therefore I think we should just move the call and the import to the other file altogether and remove this parameter. Looking at https://github.com/mozilla/pdf.js/pull/11067#discussion_r315158836, what I meant was not to move it out of `unicode.js`, but simply to move the variable above to the other file.